### ISSUE TITLE
Solar Panel Graphic Transparency

### ISSUE THREAD

**Issue by Kenosos** [Dysoch/DyTech#378]
The solar panel graphics have no transparency value giving them a very solid border overlaying terrain.

**Comment by MagicLegend** [comment]
Correct. I needed graphics quickly. Still need to do some fancy ones. I'm bad at photoshopping xD

**Comment by DreamSmith** [comment]
On images, the Secondary Solar Panel I the image/model is the wrong size.

**Comment by MagicLegend** [comment]
What do you mean by that? Is the game crashing on wrong sprite-sizes, or is the model in-game not correct with the yellow lines?
